test/handlers/ErrorHandlerSpec.scala

```diff
@@ -55,11 +56,11 @@ class ErrorHandlerSpec extends UnitSpec with Stubs with OptionValues {
 
     "standardErrorTemplate" in {
 
-      val result = errorHandler.standardErrorTemplate("Page Title", "Heading", "Message")(request).body
+      val result = errorHandler.standardErrorTemplate("title", "heading", "message")(request).body
```

Comment:
Nit: You could create constants and reference them here and on assertions.

Reply:
@MaciejRewera what do you mean with that? Sorry :)